5-AccessControl/2-call-api-groups/API/TodoListAPI/appsettings.json

```diff
@@ -1,26 +1,26 @@
 {
   "AzureAd": {
     "Instance": "https://login.microsoftonline.com/",
-    "Domain": "Enter the domain of your Azure AD tenant, e.g. 'contoso.onmicrosoft.com'",
     "TenantId": "Enter the ID of your Azure AD tenant copied from the Azure portal",
     "ClientId": "Enter the application ID (clientId) of the 'TodoListAPI' application copied from the Azure portal",
-    "ClientSecret": "Enter the Client Secret of the 'TodoListAPI' application copied from the Azure portal"
-  },
-  "GraphBeta": {
-    "BaseUrl": "https://graph.microsoft.com/v1.0",
-    "Scopes": "GroupMember.Read.All User.Read"
-  },
-  "Groups": {
-    "GroupAdmin": "Enter the objectID for GroupAdmin group copied from Azure Portal",
-    "GroupMember": "Enter the objectID for GroupMember group copied from Azure Portal"
+    "ClientSecret": "Enter the Client Secret of the 'TodoListAPI' application copied from the Azure portal",
+    "Scopes": [ "access_via_group_assignments" ],
+    "Groups": {
+      "GroupAdmin": "Enter the object ID for GroupAdmin group copied from Azure Portal",
+      "GroupMember": "Enter the object ID for GroupMember group copied from Azure Portal"
+    },
+    "MsGraph": {
```

Comment:
>MsGraph [](http://example.com/codeflow?start=5&length=7)

Look at implementing CP1 as a Graph call is involved

Reply:
Just added. Should we look into handling claims challenge in this sample or just CP1 is fine?

Reply:
handling has to be done as well

Reply:
APIs cant interact, so removed it